lib/utils.coffee

```diff
@@ -72,7 +72,12 @@ exports.isNullBuffer = (buffer) ->
 	if not Buffer.isBuffer(buffer)
 		throw new errors.ResinInvalidParameter('buffer', buffer, 'not a buffer')
 
-	return _.all buffer.toJSON(), (byte) ->
+	buffer = buffer.toJSON()
```

Comment:
It doesn't make sense to convert it to JSON just to check if it's a zero filled buffer.
```coffee
for i in buffer when i isnt 0
    return false
return true
```